src/main/java/uk/gov/pay/adminusers/queue/model/SNSMessage.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.pay.adminusers.queue.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SNSMessage {
+    @JsonProperty("Message")
```

Comment:
Is it supposed capital?

Reply:
Yep, the SNS message looks something like this:
```
{
  "Type" : "Notification",
  "MessageId" : "f03dadaf-3706-573b-8c42-f71da2a58bc2",
  "TopicArn" : "redacted",
  "Message" : "{\"sqs_message_id\":\"7ee0eef0-f989-455b-a3a8-8d84e9b4b80e\",\"service_id\":\"5e0207ee342048d4ac4d1d05d43412ff\",\"live\":true,\"resource_type\":\"dispute\",\"resource_external_id\":\"dp_1Kfmk1Hj08j2jFuBKTWzlxz5\",\"parent_resource_external_id\":\"7eufbed1265dh9mr7fc7p1e594\",\"event_date\":\"2022-03-21T15:01:53.000000Z\",\"event_type\":\"DISPUTE_CREATED\",\"event_data\":\"{\\\"fee\\\":1500,\\\"evidence_due_date\\\":1648684799,\\\"gateway_account_id\\\":\\\"528\\\",\\\"amount\\\":1000,\\\"net_amount\\\":2500,\\\"reason\\\":\\\"fraudulent\\\"}\",\"reproject_domain_object\":false}",
  "Timestamp" : "2022-03-21T15:01:55.259Z",
  ...
}
```